app/controllers/organizations_controller.rb

```diff
@@ -0,0 +1,13 @@
+class OrganizationsController < ApplicationController
+  # TODO(matthew): Are there helpers for logged_in checking? Can we run a before_action here and elsewhere to strip the
+  # current_user checks?
```

Comment:
This definitely seems like you can use a before_action, maybe ``before_action :authorize_organization`` or something.  Maybe this routes to, ``user.can_access_organization(organization)``.  Not sure.